.rhcicd/clowdapp-engine.yaml

```diff
@@ -168,9 +168,9 @@ objects:
           value: ${OB_TOKEN_USER}
         - name: OB_TOKEN_PASS
           value: ${OB_TOKEN_PASS}
-        - name: OB_MP_REST_URL
+        - name: QUARKUS_REST_CLIENT_OB_URL
           value: ${OB_MP_REST_URL}
-        - name: OB_KC_REST_URL
+        - name: QUARKUS_REST_CLIENT_KC_URL
```

Comment:
Fixed a typo in the old name here: `OB_KC_REST_URL` should have been `KC_MP_REST_URL`.